### DISCUSSION TITLE
Run multiple chats simultaneously?

### DISCUSSION THREAD

**Discussion by michox** [logancyang/obsidian-copilot#30] [Ideas]
Would it be possible to have multiple conversations about multiple notes at the same time? GPT4 can take a while to finish, and if you could for example read the text of another conversation and work on that while the first message is generating, wouldn't that be nice?

**Comment by logancyang** [comment]
This is a good idea, but would mean some significant redesign from both the UI and core logic. Currently I have a few things to build first, will check back in the near future if there's more demand for this feature. In the meantime, people who find this useful please upvote.
